onnxruntime/core/framework/session_state.cc

```diff
@@ -274,46 +316,40 @@ Status SessionState::GeneratePatternGroupCache(const std::vector<std::reference_
     auto* node = graph_viewer_->GetNode(node_plan.node_index);
     int output_start = node_index + static_cast<int>(node->InputDefs().size()) + static_cast<int>(node->ImplicitInputDefs().size());
     //allocate output
+    std::cout << "[session_state.cc, GeneratePatternGroupCache] plan for node " << node->Name() << std::endl;
```

Comment:
>std::cout << "[session_state.cc, GeneratePatternGroupCache] plan for node " << node->Name() << std::endl; [](start = 4, length = 105)

delete?

same as below.

Reply:
Ah. Thanks for catching them.

---
In reply to: [459822654](https://github.com/microsoft/onnxruntime/pull/4547#discussion_r459822654) [](ancestors = 459822654)